src/components/Table/Table.tsx

```diff
@@ -23,29 +23,44 @@ const columns: GridColDef[] = [
   },
 ];
 
-const rows = [
-  { id: 1, lastName: "Snow", firstName: "Jon", age: 35 },
-  { id: 2, lastName: "Lannister", firstName: "Cersei", age: 42 },
-  { id: 3, lastName: "Lannister", firstName: "Jaime", age: 45 },
-  { id: 4, lastName: "Stark", firstName: "Arya", age: 16 },
-  { id: 5, lastName: "Targaryen", firstName: "Daenerys", age: null },
-  { id: 6, lastName: "Melisandre", firstName: null, age: 150 },
-  { id: 7, lastName: "Clifford", firstName: "Ferrara", age: 44 },
-  { id: 8, lastName: "Frances", firstName: "Rossini", age: 36 },
-  { id: 9, lastName: "Roxie", firstName: "Harvey", age: 65 },
-];
-
 export default function DataTable() {
+  const [rows, setRows] = useState([]);
+
+  useEffect(() => {
+    const apiUrl = "https://randomuser.me/api/?results=10"; // Set the number of results to 25
```

Comment:
I believe Data Grid has a loading property. Maybe we show a loading icon while it's fetching the data.